CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ Changes to Calva.
 ## [Unreleased]
 
 ## [2.0.80] - 2020-03-07
+- Add replConnectSequences.menuSelections.projectRootDir for manually specifying the root directory in order to support jack-in for monorepo projects.
```

Comment:
Merge mistake? 😄 This should go under **Unreleased**. Also, I usually use the text and link for the issue I am fixing. So this would read:
```
- [Add option to manually specify project root directory](https://github.com/BetterThanTomorrow/calva/issues/579)
```